qt_ui/windows/basemenu/QRecruitBehaviour.py

```diff
@@ -161,24 +166,47 @@ def add_purchase_row(
     def update_available_budget(self) -> None:
         GameUpdateSignal.get_instance().updateBudget(self.game_model.game)
 
-    def buy(self, unit_type: UnitType) -> None:
+    def recruitHandler(self, recruit_type: str, unit_type: UnitType) -> None:
```

Comment:
`recruit_handler`

(I know it's not currently consistent throughout the UI but new code should follow PEP8)

Reply:
Thanks for the hint! I will keep the up with the style guide! I just reused the style you were using in the code :)

Reply:
Yeah, the UI code started pretty inconsistent :(